python/dataproc_templates/gcs/README.md

```diff
@@ -63,3 +63,76 @@ export JARS="gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar"
     --gcs.bigquery.output.mode=<append|overwrite|ignore|errorifexists> \
     --gcs.bigquery.temp.bucket.name="<temp-bq-bucket-name>"
 ```
+
+## GCS To Spanner
+
+Template for reading files from Google Cloud Storage and writing them to a
+Spanner table. It supports reading Parquet, Avro and ORC formats.
+
+## Arguments
+
+* `gcs.spanner.input.location`: GCS location of the input file
+(format: `gs://BUCKET/...`)
+* `gcs.spanner.input.format`: Input file format (one of: avro, parquet, orc)
```

Comment:
Can we support CSV and JSON as well?

I know the Java implementation only support Parquet, Avro and ORC; but it's very common to see client use CSV and JSON as well. If we add support for them, we could also backport that support to the Java templates.

Reply:
The _problem_ is that CSV files usually require additional configuration and I wanted to keep this simple. The support may be added in a different PR.

Reply:
As of now, the templates that support reading csv use sensible defaults for configuration. I agree with your point anyway, let's touch base with the rest of the team on this.

Reply:
@ppaglilla We are for now not testing these templates for text formats "csv" or "json" as it often yields to bugs.
Maybe later down the line we might choose to do it, but not for now.